server/controllers/Project.js

```diff
@@ -17,49 +17,44 @@ exports.imageUpload = (req, res) => {
 
 exports.getProject = (req, res) => {
   const { id } = req.params;
-  Project.findOne({ _id: id }, async (err, project) => {
-    if (err) {
-      res.json({
-        status: 500,
-        err
-      });
-    } else {
-      res.json({
-        status: 200,
-        project: project
-      });
-    }
-  });
+
+  Project.findOne({ _id: id })
+    .populate({ path: 'user', select: 'name' })
```

Comment:
I think this may be the best way to get the project owner name? Otherwise we would have to save the name on the model but that is a bit redundant since we have the user referenced already in the model.